hibike/hibike_tester.py

```diff
@@ -29,5 +30,15 @@ def disable(self):
 if __name__ == '__main__':
     h = Hibike()
     h.enumerate()
+    uid = h.stateQueue.get()[1][0]
```

Comment:
remove these changes. 
hibike_tester should be general, not specific to a device.